Updated docs to include INFO latencystats and LATENCY HISTOGRAM (#1733)
redis/redis-doc: commands/info.md

*   `cmdstat_XXX`: `calls=XXX,usec=XXX,usec_per_call=XXX,rejected_calls=XXX,failed_calls=XXX`

The **latencystats** section provides latency percentile distribution statistics based on the command type.

 By default, the exported latency percentiles are the p50, p99, and p999.

>  If you need to change the exported percentiles, use 'CONFIG SET latency-tracking-info-percentiles "50.0 99.0 99.9"'.

Reviewer: ```suggestion
 If you need to change the exported percentiles, use `CONFIG SET latency-tracking-info-percentiles "50.0 99.0 99.9"`.
```

>  This section requires the extended latency monitoring feature to be enabled (by default is enabled).

Reviewer: ```suggestion
 This section requires the extended latency monitoring feature to be enabled (by default it's enabled).
```

>  If you need to enable it, use 'CONFIG SET latency-tracking yes'.

Reviewer: ```suggestion
 If you need to enable it, use `CONFIG SET latency-tracking yes`.
```

For each command type, the following line is added:

> *   `latency_percentiles_usec_XXX`: `p<percentile 1>=<percentile 1 value>,p<percentile 2>=<percentile 2 value>,...`

Reviewer: ```suggestion
*   `latency_percentiles_usec_XXX: p<percentile 1>=<percentile 1 value>,p<percentile 2>=<percentile 2 value>,...`
```

Reviewer 2: why don't we quote that whole thing? i.e. it's an example of an output.

Author: @oranagra , with @yoav-steinberg 's suggestion we have it fully quoted now:
https://github.com/redis/redis-doc/pull/1733/files?authenticity_token=3Fc1AxOaZ6mXciW6ysorP5dxHiAh1vl1A-VHBn96YQvGTSnXBOV8lUj_te5XLzysq_kA12LzUKf37ffwPZotWg&file-filters%5B%5D=.md&file-filters%5B%5D=No+extension#diff-a41c08daecd2572f692ee897da8837a4da089dab5291b6ccfd6d63e9450231fcR392

*   `cmdstat_XXX`: `calls=XXX,usec=XXX,usec_per_call=XXX,rejected_calls=XXX,failed_calls=XXX`

> The **latencystats** section provides latency percentile distribution statistics based
>  on the command type.

Reviewer 2: new styling rules in README. short lines, no word wrapping
```suggestion
The **latencystats** section provides latency percentile distribution statistics based on the command type.
```
